Form/Handler.php

```diff
@@ -217,6 +217,9 @@ protected function sendMails(
         }
 
         if ($customerMailTemplatePath) {
+            // add mail text
+            $attributes['mail_text'] = $type->getCustomerMailText($form->getData());
```

Comment:
@alexander-schranz I don't know if this is the best way make the `mail_text` available in the twig template...